android/gltfio-android/src/main/java/com/google/android/filament/gltfio/AssetLoader.java

```diff
@@ -38,6 +74,7 @@
     static Constructor<IndirectLight> sIndirectLightConstructor;
     static Constructor<Skybox> sSkyboxConstructor;
 
+    /** Initializes the gltfio JNI layer. Must be called before using any gltfio functionality. */
```

Comment:
Nit, but the typical Javadoc style is to write:

```
/**
 * Single line doc.
 */
```

It helps clearly see javadoc comments in the file.